src/config/userTypes.ts

```diff
@@ -12,8 +12,8 @@ export interface IAccount {
   email: string;
   // The dietary restrictions for the user
   dietaryRestrictions: string[];
-  // The shirt size
-  shirtSize: string;
+  // The user's gender
```

Comment:
Is there a corresponding change on the api-side for this?

Reply:
https://github.com/hackmcgill/hackerAPI/pull/608